docs/diagrams/bc-registries-audit-decision-tree.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
+
+:**Error Messages from BC Registries Audit Notification Received;
+start
+:Confirm the alert by going to the 
+""Orgbook.gov.bc.ca"" url;
+if (Error Message: false) then (yes)
+  :Log into the affected OCP environment;
+  :Run ./manage deleteTopic script in orgbook configration github repo; 
+  :Run ./manage requeueOrganization script in von-bc-registries-agent-configurations github repo;
+  :Run ./manage runPipeline in von-bc-registries-agent-configurations github repo;
+  :Determine the BC registries error meesage is resloved now by logging into Orgbook.gov.bc.ca url;
+else (no)
+    :Transient issue or false positive.;
+    :Monitor for a while to see if the issues persist.;
```

Comment:
These issues are not something that need to be monitored.

Reply:
Removed the monitor for a while statement